cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -10,6 +10,9 @@
 {% if pillar['cluster_cidr'] is defined and pillar['cluster_cidr'] != "" -%}
   {% set cluster_cidr = "--cluster-cidr=" + pillar['cluster_cidr'] -%}
 {% endif -%}
+{% if pillar['service_cidr'] is defined and pillar['service_cidr'] != "" -%}
+  {% set service_cidr = "--cluster-cidr=" + pillar['service_cidr'] -%}
```

Comment:
And you need to follow the pattern with `cluster_cidr` a little more closely. I don't like this Salt file because the variables are set to nil at the top (rather than right before changing, like a ternary), but you need to add `service_cidr` up near where `cluster_cidr` is set to empty at line 2, and then here on line 14 change `service_cidr` to the new flag. (`--service-cidr=<pillar>`)